tv/components/SpeakerDisplay.tsx

```diff
@@ -0,0 +1,44 @@
+import { Session } from '../interfaces/session';
+import { Speaker } from '../interfaces/speaker';
+import Image from 'next/image';
+import SessionList from './SessionList';
+import { contentHubImageLoader } from '../utilities/contentHubImageLoader';
+
+type SpeakerDisplayProps = {
+  sessions: Session[];
+  speaker: Speaker;
+};
+
+const SpeakerDisplay = (props: SpeakerDisplayProps): JSX.Element => {
+  return (
+    <div className="w-full max-w-6xl rounded bg-white shadow-xl p-10 mx-auto text-gray-800 relative md:text-left">
+      <div className="md:flex items-center -mx-10">
+        <div className="w-full md:w-5/12 px-10 mb-10 md:mb-0">
+          <div className="relative">
+            <Image
+              loader={contentHubImageLoader}
+              src={props.speaker.photo}
+              width="400px"
+              height="400px"
+              alt="Sample"
+            />
+            <div className="border-4 border-yellow-200 absolute top-10 bottom-10 left-10 right-10 z-0"></div>
+          </div>
+        </div>
+        <div className="w-full md:w-1/2 px-10">
+          <div className="mb-10">
+            <h1 className="font-bold uppercase text-2xl mb-5">{props.speaker.name}</h1>
+            <p className="text-sm">
+              <div dangerouslySetInnerHTML={{ __html: props.speaker.description }}></div>
```

Comment:
This could be simplified to remove this `div` and keep just the parent `p` element.

Reply:
For some weird reason it does not render dangerouslySetInnerHTML= on <p> tag. Did not investigate deeper yet - do you have any ideas?

Reply:
Oh. I did not know this was an issue. Let's keep the div then.

Reply:
I successfully made it work with:
```react
<p
  className="text-sm"
  dangerouslySetInnerHTML={{ __html: props.speaker.description }}
></p>
```